### PR TITLE
TheCore Examples: EXTI and host hello world 

### PR DESCRIPTION
**Pull request by forGGe**
/cc @vadimol 
Covers #57

### CODE REVIEW

File: examples/stm32f4_exti/README.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+```
+PD9 (USART RX) - module's TX
+PD8 (USART TX) - module's RX
+```
+
+Do not forget ground.
+
+## Building
+
+Toolchain is required to build this application. The Core already have one suitable
```
Comment (RostakaGmfun) [inline examples/stm32f4_exti/README.md]: have->has

Comment (forGGe) [inline examples/stm32f4_exti/README.md]: Thanks!

File: examples/stm32f4_exti/target.hpp [hunk 2]
```diff
@@ -0,0 +1,17 @@
+// Header contains target-specific interfaces.
+
+#ifndef EXAMPLE_EXTI_TARGET_HPP_
+#define EXAMPLE_EXTI_TARGET_HPP_
+
+#include <platform/usart_bus.hpp>
+#include <platform/gpio_device.hpp>
+#include <stm32f4xx_gpio.h>
+
```
Comment (vadimol) [inline examples/stm32f4_exti/target.hpp]: is not really necessary

File: examples/stm32f4_exti/README.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+## Building
+
+Toolchain is required to build this application. The Core already has one suitable
+for this target.
+
+```
+mkdir build
+cd build
+cmake -DCMAKE_TOOLCHAIN_FILE=../../toolchains/arm-cm4-gnu.cmake
+```
```
Comment (vadimol) [inline examples/stm32f4_exti/README.md]: cmake .. -DCMAKE_TOOLCHAIN_FILE=../../toolchains/arm-cm4-gnu.cmake

**Comment by vadimol** [comment]
See comment, small mistype. 
Anyway looks good and works for me.

**Pull request merged**